.github/workflows/lint-test.yml

```diff
@@ -38,4 +38,4 @@ jobs:
       - name: Run ShellCheck
         run: |
           shellcheck --version
-          shellcheck bin/*.sh
+          shellcheck -x -e SC1091 bin/*.sh
```

Comment:
If we're excluding a rule, better practice would be to put the exclusion in the relevant bash file (I might change my mind if _all_ the scripts require this specific exclusion, but if it's just a couple, I would prefer either fixing the failing lint or excluding the rule only on the file(s) it relates to.)